notebooks/dataset_assessment_nursery.ipynb

```diff
@@ -0,0 +1,451 @@
+      "text/plain": [
+       "<Figure size 640x480 with 1 Axes>"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "from apt.risk.data_assessment.dataset_attack import DatasetAttackMembership\n",
```

Comment:
Can you please explain this cell and what is the difference between it and the previous one (they both seem to be printing the same ROC curve, so please add some text to explain, or maybe it can be removed...).

Reply:
Oh, sorry, forgot to remove the second celI, which I just used to double-check that the plot is printed only once.